dev/com.ibm.ws.classloading/src/com/ibm/ws/classloading/internal/LibertyLoader.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2014, 2023 IBM Corporation and others.
+ * Copyright (c) 2014, 2024 IBM Corporation and others.
```

Comment:
```suggestion
 * Copyright (c) 2014, 2023 IBM Corporation and others.
```
Nothing was changed so shouldn't update the copyright.